src/matcha_ml/runners/base_runner.py

```diff
@@ -131,9 +131,12 @@ def _check_matcha_directory_exists(self) -> None:
             )
             raise typer.Exit()
 
-    def _apply_terraform(self) -> None:
+    def _apply_terraform(self, msg: str = "") -> None:
         """Run terraform apply to create resources on cloud.
 
+        Args:
+            msg (str) : Message to display. Default is empty string.
```

Comment:
This gives the impression that it's the entire message that someone would need to pass, otherwise nothing is shown. It probably wants some clarification that this is actually the name for the group of resources being provisioned